config/rbac/submariner-gateway/service_account.yaml

```diff
@@ -3,3 +3,8 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: submariner-gateway
+  annotations:
+    kubernetes.io/enforce-mountable-secrets: "true"
+secrets:
+  - name: submariner-broker-secret
+  - name: submariner-ipsec-psk
```

Comment:
I see you're listing mounted secrets here but not for the others. How come?

Reply:
This is the only SA that accesses secrets other than the Kubernetes-provisioned secret (which is allowed by default).

Reply:
Ah, no, the Lighthouse agent does this too...

Reply:
So if specific `secrets` are not specified, then this will only allow the default K8s-provisoned secret to be mounted.

I wonder how it works with `automountServiceAccountToken: false` ...

Reply:
I think that’s orthogonal: this setting determines which secrets are allowed to be mounted, the automount setting determines whether the default provisioned secret is mounted automatically.